docs/design/pop-precision-tuning.md

```
@@ -14,79 +16,75 @@ Mixed-precision programming is a manual, error-prone process. Developers must de
-**Subtraction** `z = x - y`:
-```text
-nsb_F(z) >= min(nsb(x) + ufp(x), nsb(y) + ufp(y)) - ufp(z) - carry(-)
-```
+| Operation | Forward Transfer |
+|-----------|-----------------|
+| `z = x + y` | `nsb(z) = ufp(z) - min(lsb(x), lsb(y)) + 1 + carry` |
+| `z = x - y` | Same as addition |
+| `z = x * y` | `nsb(z) = nsb(x) + nsb(y) + carry` |
+| `z = x / y` | `nsb(z) = nsb(x) + nsb(y) + carry` |
+| `z = -x` | `nsb(z) = nsb(x)` |
+| `z = |x|` | `nsb(z) = nsb(x)` |
```

markdownlint-cli2 (0.21.0)

[warning] Table column count
Expected: 2; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)